index.yaml

```diff
@@ -1856,6 +1856,39 @@ entries:
     - assets/rancher-grafana/rancher-grafana-6.6.401.tgz
     version: 6.6.401
   rancher-istio:
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-kiali-server-crd=100.0.0+up1.35.0
```

Comment:
@brendarearden  Just need a confirmation: the crd chart version can be different from the main chart version, so the value here is valid, right?

Reply:
correct!